### ISSUE TITLE
Marker Tiles Clipped

### ISSUE THREAD

**Issue by stevehorn** [TileStache/TileStache#151]
I am playing around with the tile server using the mapnik plugin and am seeing some problems viewing the tiles through the modestmaps previewer. I'm not sure where at in the chain the problem lies (modestmaps, tilestache, mapnik?). Anyone else experienced this, or able to tell me how I might troubleshoot?

![markers_big](https://f.cloud.github.com/assets/226419/1018331/96874ad0-0c26-11e3-8847-d88a77419a1c.png)

**Comment by zhaochunliang** [comment]
Hi stevehorn, have you found the solution about this issue ? I met similar issue, could you please share your method? thanks.

**Comment by stevehorn** [comment]
@zhaochunliang I have some notes I took from that time and I don't think I was ever able to solve this problem using TileStache. Here is what I wrote down:

After more research this is a well known problem that is solved by "marker buffering" or "metaTiling". 

In the end I was able to get the desired functionality using [TileCache](http://tilecache.org/).

Hope this helps.

**Comment by zhaochunliang** [comment]
@stevehorn, Yes, with TileStache I tried to set  "metatile":{"rows":16,"columns":16,"buffer":128}  in tilestache config file and set the avoid-edges="true" for Point style, I think the problem should disappear, but it does not.  
![qq 20150802140523](https://cloud.githubusercontent.com/assets/9548981/9024954/aeaa2876-391f-11e5-847f-0618e365d166.png)

Do you find the root cause in code of Tilestache ?
